### PR TITLE
Add links to DALI related GTC'23 talks

### PR DESCRIPTION
**Pull request by JanuszL**
## Category:
**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- adds links to DALI related GTC'23 talks

### Affected modules and functionalities:
- README

### Key points relevant for the review:
- NA

### Tests:

- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [x] Other
    - README
- [ ] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

**Review by mzient — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -216,6 +218,10 @@ Additional Resources
 .. _talkIntro2022: https://www.nvidia.com/en-us/on-demand/session/gtcspring22-s41443/
 .. |talkAdvanced2022| replace:: event
 .. _talkAdvanced2022: https://www.nvidia.com/en-us/on-demand/session/gtcspring22-s41442/
+.. |talkGeospatial2023| replace:: event
+.. _talkGeospatial2023: https://www.nvidia.com/gtc/session-catalog/?search=S51796&tab.catalogallsessionstab=16566177511100015Kus#/session/1666647534514001Pu4y
+.. |breakoutDALITRITON| replace:: event
+.. _breakoutDALITRITON: https://register.nvidia.com/flow/nvidia/gtcspring2023/attendeeportal/page/sessioncatalog/session/1670621650376001XfVr
```
[the remark this answers is not in the document]
Comment (JanuszL) [reply README.rst]: Fixes

**Review by klecki — APPROVED** [review]

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

**Pull request merged**